manifests/administration.pp

```diff
@@ -30,14 +30,43 @@
       value  => '1',
     }
   }
+  file { '/etc/systemd/system/disable_thp.service':
+    ensure  => file,
+    mode    => '0644',
+    owner   => 'root',
+    group   => 'root',
+    content => file('redis/service_files/disable_thp.service'),
+  }
```

Comment:
We already depend the systemd module. Perhaps it should use `systemd::unit_file` instead?

Reply:
Done